search-service/src/main/kotlin/com/egm/stellio/search/service/EntityEventListenerService.kt

```diff
@@ -128,11 +132,18 @@ class EntityEventListenerService(
         temporalEntityAttributeService.getForEntityAndAttribute(
             entityId, expandedAttributeName, datasetId
         ).zipWhen {
+            val instanceId = "urn:ngsi-ld:Instance:${UUID.randomUUID()}".toUri()
```

Comment:
Hmm, I see the point, but I don't really like having this `instanceId` extracted in a few places... any other option?